pydarn/plotting/fan.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett, Marina Schmidt
+"""
+Fan plots, mapped to AACGM coordinates
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import aacgmv2
+import datetime as dt
+import os
+
+from pydarn import PyDARNColormaps
+from typing import List
+from matplotlib import ticker, cm, colors
+
+class fan():
+    # TODO: Add class documentation
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - plot_fan()\n"\
+                "   - return_beam_pos()\n"
+
+    @classmethod
+    def plot_radar_fov(cls, dmap_data: List[dict], scan_index: int = 0,
```

Comment:
whoops, wrong comment box.

@aburrell @asreimer @billetd thoughts? 

When I see `radar_fov` I just think the fov plot, not the data.

Reply:
I'm open to changing this. A seperate method for just the FOV would also be pretty simple to add.

Reply:
I think I agree, it would be SUPER useful to me going forward if I could use a method in pydarn to grab just the FOV easily.

Reply:
I think FOV plotter would be good to just plot a radar region. Not in this PR though haha

Reply:
Yes, put an issue in for that!  Also, it's a thing I use all the time.  I agree with @mts299 about the name, because I now realise in that in the back of my mind this is the type of plot I was visualising for this PR.

Reply:
Updated the name to `plot_fan`. Dedicated FOV plotter will come in a seperate PR.